.travis.yml

```diff
@@ -6,6 +6,7 @@ rvm:
   - 2.5.8
   - 2.6.6
   - 2.7.2
+  - 3.0.0
```

Comment:
From the [CI logs](https://travis-ci.org/github/utkarsh2102/rubocop-packaging/jobs/751667508#L245):
```
$ ruby --version
ruby 3.0.0preview2 (2020-12-08 master d7a16670c3) [x86_64-linux]
```

Hm, so it looks like that ruby3.0 isn't available yet. Probably I'll wait and re-trigger the build in some days!? 🤔 
However, I don't anticipate anything breaking w/ ruby3.0, honestly. I'd be surprised if something does, I guess.

Reply:
Weeehooooo! Allll goooood! 🎉 🌮 🎈 🍾 🍰 
ruby3.0 and ruby-head all green and clean! \o/